scripts/rebuild_languages.py

```diff
@@ -33,104 +35,120 @@ def downloadSheet(url):
     return list(reader)
 
 
-languageSpreadsheetURL = (
-    "https://docs.google.com/"
-    "spreadsheets/d/1woTU8dZCHJh7yvdk-N1kgQBUj4Sn3SdRsbKgn6ltJQs/"
-)
+def main(wip):
+    languageSpreadsheetURL = (
+        "https://docs.google.com/"
+        "spreadsheets/d/1woTU8dZCHJh7yvdk-N1kgQBUj4Sn3SdRsbKgn6ltJQs/"
+    )
 
-rows = downloadSheet(languageSpreadsheetURL + "export?format=csv")
+    rows = downloadSheet(languageSpreadsheetURL + "export?format=csv")
 
-numHeaders = 5
-headers = rows[:numHeaders]
-assert headers[0][0] == "Go to Documentation", headers[0][0]
-assert headers[1][2] == "English", headers[1][2]
-assert headers[2][2] == "English", headers[2][2]
-assert headers[3][2] == "en", headers[3][2]
+    numHeaders = 5
+    headers = rows[:numHeaders]
+    assert headers[0][0] == "Go to Documentation", headers[0][0]
+    assert headers[1][2] == "English", headers[1][2]
+    assert headers[2][2] == "English", headers[2][2]
+    assert headers[3][2] == "en", headers[3][2]
 
-rows = rows[numHeaders:]
+    rows = rows[numHeaders:]
 
-startColumn = 2
+    startColumn = 2
 
-languages = []
-languageStrings = {}
+    languages = []
+    languageStrings = {}
 
-for columnIndex in range(startColumn, len(headers[1])):
-    languageInEnglish = headers[1][columnIndex]
-    languageInLanguage = headers[2][columnIndex]
-    languageCode = headers[3][columnIndex]
-    languageStatus = headers[4][columnIndex]
-    assert languageCode
-    if not languageStatus.strip():
-        continue
+    for columnIndex in range(startColumn, len(headers[1])):
+        languageInEnglish = headers[1][columnIndex]
+        languageInLanguage = headers[2][columnIndex]
+        languageCode = headers[3][columnIndex]
+        languageStatus = headers[4][columnIndex]
+        assert languageCode
+        if not languageStatus.strip():
+            continue
 
-    languages.append(
-        dict(
-            code=languageCode,
-            langEn=languageInEnglish,
-            langLang=languageInLanguage,
-            status=languageStatus,
+        languages.append(
+            dict(
+                code=languageCode,
+                langEn=languageInEnglish,
+                langLang=languageInLanguage,
+                status=languageStatus,
+            )
         )
+
+        languageStrings[languageCode] = strings = {}
+
+        for row in rows:
+            if not wip and row[0] == "WIP":
```

Comment:
```suggestion
            if wip or "WIP" not in row[0]:
```
1. make the meaning clearer
2. allow the comment collumn to contain info beyond WIP

We could also say "WIP" must be at the beginning of the cell, if general containment is too loose.

Reply:
I am closing this, as we solved it a little bit different.